internal/cli/kraft/run/utils.go

```diff
@@ -459,3 +459,66 @@ func (opts *RunOptions) parseEnvs(_ context.Context, machine *machineapi.Machine
 
 	return nil
 }
+
+// GetArgs returns all the arguments of run command as an array of strings.
+func (opts *RunOptions) GetArgs() []string {
+	args := []string{}
+	if opts.Detach {
+		args = append(args, "--detach")
+	}
+	if opts.DisableAccel {
+		args = append(args, "--disable-acceleration")
+	}
```

Comment:
Check out: https://pkg.go.dev/reflect#StructTag.Get

This will simplify getting the flag name.

Reply:
Got it 👍🏽